Spigot-Server-Patches/0249-Attempt-2-biome-replacement.patch

```diff
@@ -0,0 +1,60 @@
+From b1a83ff1969c49ff83bcd3a0f3dbcb2220b584c6 Mon Sep 17 00:00:00 2001
+From: June <junehanabi@gmail.com>
+Date: Wed, 18 Oct 2017 22:07:27 -0500
+Subject: [PATCH] Attempt 2 biome-replacement
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index a9637da0d..e751b5718 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -310,31 +310,30 @@ public class PaperConfig {
+     public static Map<String, String> biomeReplacement = new HashMap<String, String>();
+ 
+     private static void biomeReplacement() {
+-        List<Map<String, String>> biomes = getMap("biome-replacement", new ArrayList<>());
++        List<Map> biomes = getMap("biome-replacement", new ArrayList<>());
```

Comment:
....no. roughly:

```java
// PaperConfig
    public static ConfigurationSection biomeReplacements;
    private static void biomeReplacements() {
        biomeReplacements = config.getConfigurationSection("biome-replacements");
    }

// and then in Biomes...
s = PaperConfig.biomeReplacements.get(s, s);
```

Reply:
Thank you so much! I'm testing it out now.